src/components/Preferences/Preferences.js

```diff
@@ -125,8 +125,8 @@ const AnimatedPreferences = ({ opened, ...props }) => {
         /* eslint-disable react/prop-types */
         (({ opacity, enterProgress, blocking }) => (
           <AnimatedWrap
-            smallView={smallView}
             style={{
+              zIndex: smallView ? 2 : 5,
```

Comment:
Just wanted to confirm that the `2` is to keep the menu above the preferences during the swipe interaction? If so, maybe we could add a comment to remember it :spiral_notepad: